app/mm-redux/types/config.ts

```diff
@@ -9,6 +9,7 @@ export type Config = {
     AndroidAppDownloadLink: string;
     AndroidLatestVersion: string;
     AndroidMinVersion: string;
+    AppCustomURLScheme: string;
```

Comment:
We have added the custom url scheme as server property. The server will redirect to this URL upon completing the login.

We are passing it to the client-side and the client will send the same to the server and we are verifying it on the server-side during the login initiation.

https://mattermost.atlassian.net/browse/MM-29644, checks have been implemented for both web and mobile to not redirect to unknown urls.

Reply:
I don't see the point of doing this. If the scheme is not set in the app it won't work.

For example if I set the server config to use the scheme `myapp` then the deeplink will do nothing or open another app if it can open that scheme, to me this is very error prone and allows the admin to mess it up

Reply:
True, I agree with the admin messing it up ..  but the server needs a way to not redirect to unknown redirect URLs so I thought making it a config property is a way to make sure of that.

Reply:
I'm 5/5 on removing it and having the server prwvent the redirect if the scheme is not `mattermost://`

Reply:
Although I have considered this, it would not work for custom compiled apps and will redirect to our app instead of the client's app.

Reply:
Says who? Custlm compiled apps can change the scheme, but they can just add a secondary one if we modify the fastlane script to add instead of replacing and then we can just document it. Again 5/5 on this

Reply:
Sorry, I mean it would not work without changing the value on the server-side but wouldn't it again have a chance to be messed by the admin?

Reply:
It wouldn't as whoever compiles the mobile app will just add another scheme, the default mattermost scheme will always be present.